info.bibletime.BibleTime.json

```
@@ -0,0 +1,91 @@
+{
+  "id": "info.bibletime.BibleTime",
+  "branch": "stable",
+  "sdk": "org.kde.Sdk",
+  "runtime": "org.kde.Platform",
+  "runtime-version": "5.11",
+  "command": "bibletime",
+  "rename-icon": "bibletime",
+  "rename-desktop-file": "bibletime.desktop",
+  "finish-args": [
+    "--socket=x11",
+    "--share=ipc",
+    "--socket=wayland",
+    "--share=network",
+    "--persist=.bibletime",
```

```suggestion
    "--persist=.bibletime",
    "--device=dri",
```
You need to allow GPU usage in order for Qt to be able to use this acceleration otherwise you get errors on startup.

---

Thanks. I changed to using "dri".